chrome/content/coniks/coniks.js

```diff
@@ -0,0 +1,90 @@
+this.EXPORTED_SYMBOLS = ["coniks"];
+
+var { interfaces: Ci, utils: Cu, classes: Cc } = Components;
+
+Cu.import("resource:///modules/imServices.jsm");
+Cu.import("resource:///modules/imXPCOMUtils.jsm");
+Cu.import("resource:///modules/jsProtoHelper.jsm");
+Cu.import("chrome://otr/content/coniks/context.js");
+Cu.import("chrome://otr/content/helpers.js");
+Cu.import("chrome://otr/content/otr.js");
+
+function accountname(acc) {
+  let account = acc.normalizedName;
+  let protocol = acc.protocol.normalizedName;
+  return account+"@"+protocol;
+}
+
+var coniks = {
+  _isEnable: true,
```

Comment:
This should be `false` by default for now.

And to enable it, you would need to manually access the config editor and set `extensions.otr.coniks.enabled` to `true`.

So, when `init()` is called, you should,

```
coniks.setPrefs();
coniks.isEnabled = coniks.prefs.getBoolPref('enabled');
if (!coniks.isEnabled)
  return;
```

see ui.js for similar.